knowledge_repo/app/routes/posts.py

```diff
@@ -172,3 +177,33 @@ def render_legacy():
 def about():
     """Renders about page. This is the html version of REAMDE.md"""
     return render_template("about.html")
+
+
+@blueprint.route('/ajax/post/download', methods=['GET'])
+@PageView.logged
+def download():
+    "Downloads resources associated with a post."
+
+    path = request.args.get('post')
```

Comment:
personal preference, I don't like to assign to variables if they're not used so in this case I'd prefer

post = current_repo.post(request.args.get('post'))